types/hopscotch/index.d.ts

```diff
@@ -1,9 +1,10 @@
 // Type definitions for Hopscotch v0.2.5
 // Project: http://linkedin.github.io/hopscotch/
-// Definitions by: Tim Perry <https://github.com/pimterry>
+// Definitions by: Tim Perry <https://github.com/pimterry> , Aurimas <https://github.com/Aurimas1>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare type CallbackNameNamesOrDefinition = string | string[] | (() => void);
+declare type placmentTypes = 'top' | 'bottom' | 'right' | 'left';
```

Comment:
Typo, `placment` should be `placement` (or maybe position?). I don't think you need the `Type` suffix either.

Reply:
I fix typo, but I like suffix, because just placement do not say anything